apps/documents/assets/__tests__/react_documents_render.jest.jsx

```diff
@@ -1,21 +0,0 @@
-// checking for simple element to see if component is rendered as interim solution
-import '../../../../__mocks__/jsDomMock'
```

Comment:
removed as hoping to redo all the tests and add the additional ones from mB in another PR, just not working yet https://github.com/liqd/adhocracy-plus/pull/2105